README.md

```diff
@@ -37,7 +37,7 @@ Join `#buildkit` channel on [Docker Community Slack](https://dockr.ly/comm-slack
 
 > **Note**
 >
-> [BuildKit has been integrated to `docker build` since Docker 18.09](https://docs.docker.com/develop/develop-images/build_enhancements/).
+> [BuildKit is used by the default implementation of `docker build` since Docker Engine 23.0](https://docs.docker.com/build/architecture/).
```

Comment:
```suggestion
> In Docker Engine version 23.0 and later, `docker build` [uses BuildKit by default](https://docs.docker.com/build/architecture/).
```

Reply:
Historical information about 18.09 can be still preserved in somewhere.

Reply:
(And eventually we should have the full documentation to explain `DOCKER_BUILDKIT` vs `docker buildx` vs BuildKit  and their histories...)
- https://github.com/moby/buildkit/issues/1214

Reply:
I'm not sure what version 22.03 is supposed to refer to -- did you mean 20.10?

Reply:
Yes, it was a typo 😅

Reply:
I'm not 100% on board with this wording, as I want to make it clear that `docker build` is implemented by Buildx, but without having to explain that in depth (since users can click through).

We can't really say `docker build` uses BuildKit by default as it doesn't anymore -- we actually made it forget how to use BuildKit via the REST API. Instead, CLI versions 23.0+ **replace the `docker build` implementation with `docker buildx build`** when `DOCKER_BUILDKIT=1` (by default) and fall back to the classic `docker build` implementation/classic builder when `DOCKER_BUILDKIT=0`.

`DOCKER_BUILDKIT=1` without Buildx installed is now an error.

---

I think my original phrasing was closer to communicating that (and again, more becomes clear when you click through) -- can you think of a better way to reconcile these two together @dvdksn?

Reply:
I don't like to call it the "default **implementation**" of docker build.

> We can't really say docker build uses BuildKit by default as it doesn't anymore

It's still true that the docker build command uses buildkit by default, it's just using buildx to get there. How about changing to:

```suggestion
> `docker build` [uses Buildx by default](https://docs.docker.com/build/architecture/) since Docker Engine 23.0.
```

Reply:
Maybe we can cut the baby in half with "uses Buildx and BuildKit by default"?

Reply:
SGTM!